proposed/http-middleware/middleware-meta.md

```diff
@@ -101,28 +110,23 @@ request and response being passed to the middleware.
 
 The primary downside of this interface is that the while the interface itself is
 a callable, there is currently no way to type hint a closure in a similar way.
-This may be resolved in the future with [functional interfaces][php-functional].
-
-[php-functional]: https://wiki.php.net/rfc/functional-interfaces
 
 ### 4.2 Single Pass (Lambda)
 
 The other approach to middleware is much closer to [StackPHP][stackphp] style
 and is defined as:
 
-
 ```
-fn(request, next): response
+fn(request, frame): response
```

Comment:
In my opinion it is not obvious to the reader what `frame` means and why it is used here. It is the very first time this word is used for that parameter and it is never used again – neither within this document nor in proposed/http-middleware/middleware.md.

I would stick with `next`. Double-pass frameworks and Laravel use that term; Guzzle and StackPHP does not have that parameter. I would not use `delegate` at this point, the naming will be explained in a following section.